drivers/ft5x06/ft5x06_touch_dev.c

```diff
@@ -55,8 +55,16 @@ uint8_t _ft5x06_touches(const touch_dev_t *touch_dev, touch_t *touches, size_t l
     uint8_t ret;
     ft5x06_read_touch_count(dev, &ret);
 
+    uint8_t count_max;
+    if (dev->params.type == FT5X06_TYPE_FT6X06 || dev->params.type == FT5X06_TYPE_FT6X36) {
+        count_max = FT6XX6_TOUCHES_COUNT_MAX;
+    }
+    else {
+        count_max = FT5X06_TOUCHES_COUNT_MAX;
+    }
+
```

Comment:
Maybe add an internal function to get the number of max touches and avoid a duplicate code.

Reply:
Factored out in 7d49dec8ced116282e798748cbc7e816fbec4738